roles/sap_ha_install_anydb_ibmdb2/tasks/passwordless_ssh.yml

```diff
@@ -55,26 +59,33 @@
         - "/db2/db2{{ sap_ha_install_anydb_ibmdb2_sid | lower }}/.ssh/anydb_ibmdb2_hadr_{{ inventory_hostname_short }}"
         - "/db2/db2{{ sap_ha_install_anydb_ibmdb2_sid | lower }}/.ssh/anydb_ibmdb2_hadr_{{ inventory_hostname_short }}.pub"
 
-    - name: SAP HA AnyDB - IBM Db2 HADR - Get SSH fingerprint of SAP AnyDB Primary node
+    # Reasons for noqa:
+    # - no-changed-when: Shell if logic to handle SSH Keys between hosts to align with IBM Db2 documentation
+    - name: SAP HA AnyDB - IBM Db2 HADR - Get SSH fingerprint of SAP AnyDB Primary node # noqa no-changed-when
       ansible.builtin.shell: |
         ssh-keyscan -t rsa {{ sap_ha_install_anydb_ibmdb2_hostname_primary }}
       register: __sap_ha_install_anydb_ibmdb2_primary_ssh_fingerprint
       when: sap_ha_install_anydb_ibmdb2_hostname_primary == inventory_hostname_short
 
-    - name: SAP HA AnyDB - IBM Db2 HADR - Get SSH fingerprint pf SAP AnyDB Secondary node
+    # Reasons for noqa:
+    # - no-changed-when: Shell if logic to handle SSH Keys between hosts to align with IBM Db2 documentation
+    - name: SAP HA AnyDB - IBM Db2 HADR - Get SSH fingerprint pf SAP AnyDB Secondary node # noqa no-changed-when
       ansible.builtin.shell: |
         ssh-keyscan -t rsa {{ sap_ha_install_anydb_ibmdb2_hostname_secondary }}
       register: __sap_ha_install_anydb_ibmdb2_secondary_ssh_fingerprint
       when: sap_ha_install_anydb_ibmdb2_hostname_secondary == inventory_hostname_short
 
-
-    - name: SAP HA AnyDB - IBM Db2 HADR - Get contents of SSH Public Key for SAP AnyDB Primary node
+    # Reasons for noqa:
+    # - no-changed-when: Shell if logic to handle SSH Keys between hosts to align with IBM Db2 documentation
+    - name: SAP HA AnyDB - IBM Db2 HADR - Get contents of SSH Public Key for SAP AnyDB Primary node # noqa no-changed-when
       ansible.builtin.shell: |
```

Comment:
This can be replaced by Ansible using the `file` lookup or the `slurp` module.

Example using the `file` lookup:
```      ansible.builtin.set_fact:
__sap_ha_install_anydb_ibmdb2_secondary_pubkey: "{{ lookup('ansible.builtin.file', '/root/.ssh/anydb_ibmdb2_hadr_{{ inventory_hostname_short }}.pub') }}"
```
Example using the `slurp` module:
https://github.com/sap-linuxlab/community.sap_install/blob/bd93292e1dea7b781e5ba6faa4fc77bf5588639a/roles/sap_hana_preconfigure/tasks/RedHat/generic/enable-tsx.yml#L4
and
https://github.com/sap-linuxlab/community.sap_install/blob/bd93292e1dea7b781e5ba6faa4fc77bf5588639a/roles/sap_hana_preconfigure/tasks/RedHat/generic/enable-tsx.yml#L19